components/src/core/UserMenu/UserMenu.tsx

```diff
@@ -62,6 +62,29 @@ export const UserMenu: React.FC<UserMenuProps> = (props) => {
     const defaultClasses = useStyles(theme);
     const [anchorEl, setAnchorEl] = useState(null);
 
+    const closeMenu = useCallback(() => {
+        onClose();
+        setAnchorEl(null);
+    }, [onClose]);
+
+    // Add closeMenu function to each item that has an onClick function.
+    useEffect(() => {
+        for (const group of menuGroups) {
+            for (const item of group.items) {
+                const onClick = item.onClick;
+                if (onClick) {
+                    item.onClick = (): void => {
+                        onClick();
+                        closeMenu();
+                    }
+                } else {
+                    item.InfoListItemProps = { ripple: false, ...item.InfoListItemProps };
```

Comment:
should we instead update the InfoListItem to not show the ripple unless there is a onClick event? Like we do with hover?

Reply:
I agree, I made the MUI ListItem `button` prop dependent on the the InfoListItem's `ripple` and `onClick` being set.   It doesn't make sense for non-clickable items to have a ripple effect for now reason, right?